### PR TITLE
prow: Update cherry pick comment

### PR DESCRIPTION
**Pull request by justaugustus**
A slight revision to #13137.
Signed-off-by: Stephen Augustus <saugustus@vmware.com>

### CODE REVIEW

File: prow/plugins/config.go [hunk 1]
```diff
@@ -731,11 +731,12 @@ func (c *Configuration) setDefaults() {
 		c.CherryPickUnapproved.BranchRegexp = `^release-.*$`
 	}
 	if c.CherryPickUnapproved.Comment == "" {
-		c.CherryPickUnapproved.Comment = `This PR is not for the master branch but does not have the ` + "`cherry-pick-approved`" + `  label. Adding the ` + "`do-not-merge/cherry-pick-not-approved`" + `  label.
+		c.CherryPickUnapproved.Comment = `This cherry pick PR is for a release branch and has not yet been approved by the Patch Release Team.
+Adding the ` + "`do-not-merge/cherry-pick-not-approved`" + `  label.
 
-To approve the cherry-pick, please ping the *kubernetes/patch-release-team* in a comment when ready.
+To merge this cherry pick, please ping the *kubernetes/patch-release-team* in a comment, **after** it have been approved by the relevant OWNERS.
```
Comment (cblecker) [inline prow/plugins/config.go]:
```suggestion
To merge this cherry pick, please ping the `@kubernetes/patch-release-team` in a comment, **after** it have been approved by the relevant OWNERS.
```

Comment (justaugustus) [reply prow/plugins/config.go]: @cblecker -- will the backticks still trigger the GH mention? I wasn't sure.

Comment (justaugustus) [reply prow/plugins/config.go]: Ahhhhh, I see above that it doesn't. Will fix that up!

Comment (justaugustus) [reply prow/plugins/config.go]: Fixed!

**Comment by cblecker** [comment]
Actually, I think the cherry pick plugin is used outside of Kubernetes, so we can't make this message more specific without offering it as a configurable option.

/hold

**Comment by justaugustus** [comment]
@cblecker -- Oh no! The original, more ambiguous phrasing makes more sense now. 
Will look into this... "later".

**Comment by stevekuznetsov** [comment]
Agreed it should be in config, not code. Not sure why this "default" is in there in the first place.

File: prow/plugins/config.go [hunk 2]
```diff
@@ -731,11 +731,12 @@ func (c *Configuration) setDefaults() {
 		c.CherryPickUnapproved.BranchRegexp = `^release-.*$`
 	}
 	if c.CherryPickUnapproved.Comment == "" {
-		c.CherryPickUnapproved.Comment = `This PR is not for the master branch but does not have the ` + "`cherry-pick-approved`" + `  label. Adding the ` + "`do-not-merge/cherry-pick-not-approved`" + `  label.
+		c.CherryPickUnapproved.Comment = `This cherry pick PR is for a release branch and has not yet been approved by the Patch Release Team.
+Adding the ` + "`do-not-merge/cherry-pick-not-approved`" + ` label.
 
-To approve the cherry-pick, please ping the *kubernetes/patch-release-team* in a comment when ready.
+To merge this cherry pick, please ping the ` + "`@kubernetes/patch-release-team`" + ` in a comment, **after** it has been approved by the relevant OWNERS.
```
Comment (fejta) [inline prow/plugins/config.go]: This will still trigger pings when the many people use email to reply to this comment

**Comment by justaugustus** [comment]
Superseded by #14275.
/close

**Pull request closed without merging**